### PR TITLE
Acquisuite dataServer - Minor changes and bugs fixed.

### PR DESCRIPTION
**Pull request by jackrwoods**
Although the code compiled and ran, BodyParser was returning undefined request bodies. These changes have employed the urlencoded body parser for data collection.

### CODE REVIEW

**Comment by jackrwoods** [comment]
Also, you can see the output by ssh-ing into the ec2 instance and navigating to apps/SeniorCapstone/Source/acquisuite-data/postData.txt

**Comment by DSchroederOSU** [comment]
Jack. The req.body was empty because the route was not "getting" called from anywhere. There was no $http.post to create a req body.

Am I correct in my assumption?

**Comment by jackrwoods** [comment]
I was generating HTTP requests with [hurl.it](http://hurl.it), so the route was getting called. Requests were also being logged to the console- so I don't think that was the problem.

I believe there was a formatting issue with my artificial post requests. I was only including text in the body of the request (no headers or other metadata), so the BodyParser was not interpreting the requests correctly.

As it stands right now, it works!

File: Source/dataServer.js [hunk 1]
```diff
@@ -26,6 +26,11 @@ db.once('open', function() {
 // log every request to the console
 app.use(morgan('dev'));
 
+// Parse post bodies
+app.use(bodyParser.urlencoded({
+  extended: true;
+}));
+
```
[the remark this answers is not in the document]
Comment (jackrwoods) [reply Source/dataServer.js]: Ok, will change!

**Comment by jackrwoods** [comment]
The changes have been made and the code is currently deployed on the EC2. This is ready for review + merge.

**Pull request merged**